File: JS Advanced/Basic Syntax - Lab/mathOperations.js
function Solve(firstNumber, secondNumber, operator) {

    let result;
    switch (operator) {
        case '+':
            result = Number(firstNumber) + Number(secondNumber);
            break;
        case '-':
            result = Number(firstNumber) - Number(secondNumber);
            break;
        case '/':
            result = Number(firstNumber) / Number(secondNumber);
            break;
        case '*':
            result = Number(firstNumber) * Number(secondNumber);
            break;
        case '**':
            result = Number(firstNumber) ** Number(secondNumber);
            break;
        case '%':
            result = Number(firstNumber) % Number(secondNumber);
            break;
    }

    return result;
}

console.log(Solve('5', '6', '+'));

File: C# OOP/Polymorphism - Lab/MathOperations/MathOperations.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MathOperations
{
    public class MathOperations
    {
        public int Add(int firstNumber, int secondNumber)
        {
            var result = firstNumber + secondNumber;

            return result;
        }
        public double Add(double firstNumber,double secondNumber,double thirdNumber)
        {
            var result = firstNumber + secondNumber + thirdNumber;

            return result;
        }
        public decimal Add(decimal firstNumber,decimal secondNumber,decimal thirdNumber)
        {
            var result = firstNumber + secondNumber + thirdNumber;

            return result;
        }
    }
}
